third-party/realdds/doc/discovery.md

```diff
@@ -64,16 +64,26 @@ The message format is a [flexible message](../include/realdds/topics/flexible/)
 }
 ```
 
-| Field        | Purpose                                                |
-|--------------|--------------------------------------------------------|
-| name         | The name of the device, as it would show in the Viewer |
-| serial       | The device serial number                               |
-| product-line | The type of device (D400, L500, etc.)                  |
-| topic-root   | The path to the root topic for the device              |
-| multicast-ip | IP address on which topics will be multicast           |
+| Field | Purpose | rs2_camera_info |
+| ---- | ---- | ---- |
+| **name** | The name of the device | `RS2_CAMERA_INFO_NAME` |
+| serial | The device serial number | `RS2_CAMERA_INFO_SERIAL_NUMBER` |
+| product-line | The type of device (D400, L500, etc.) | `RS2_CAMERA_INFO_PRODUCT_LINE` |
+| **topic-root** | The path to the root topic for the device | `RS2_CAMERA_INFO_PHYSICAL_PORT` |
+| fw-version | The version of the software currently on the device | `RS2_CAMERA_INFO_FIRMWARE_VERSION` |
 
 All are optional except `name` and `topic-root`. Any fields not shown above will be ignored.
 
+The fields above are constants and not expected to change between device restarts! They should likewise remain the same in recovery mode.
```

Comment:
I am not sure that serial remains the same for recovery device. Pretty sure that name also changes.

Reply:
This is not for legacy devices, and the adapter will have to behave in a way that aligns, in my next PR likely.
I may change the name requirement (just for the adapter), but for DDS-native cameras, the serial and name requirements have already been agreed on.